includes/api/ApiQueryWatchlist.php

```diff
@@ -270,7 +270,8 @@ private function extractRowInfo( $row ) {
 		if ( $this->fld_user || $this->fld_userid ) {
 
 			if ( $this->fld_user ) {
-				$vals['user'] = User::getUsername( $row->rc_user, $row->rc_user_text );
+				$userIp = RecentChange::extractUserIpFromRow( $row );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$userIp" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app